doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -712,9 +712,9 @@ And this maximum is:
 
```

Comment:
I don't see how it is weird, your computation will start from the children and end in the root with the final number corresponding to the minimum base allowed for this tree.